### ISSUE TITLE
Publish was successful but npm install doesn't work

### ISSUE THREAD

**Issue by adibwoy** [npm/npm#20574]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] npm is producing incorrect or undesirable behavior.
  - [x] Other (_see below for feature requests_):

#### What's going wrong?
We published 0.11.17 of `@clr/angular`, `@clr/ui`, `@clr/icon` and got a confirmation message from npm:
![image](https://user-images.githubusercontent.com/1426805/39872277-754565ee-5435-11e8-8014-7b0b6c2854ce.png)

While searching for the package on npmjs.org we see this:
![image](https://user-images.githubusercontent.com/1426805/39872326-9c500f36-5435-11e8-8a54-d2f9e62bd217.png)
![image](https://user-images.githubusercontent.com/1426805/39872347-adeead6a-5435-11e8-9b82-10c687d4f35e.png)

but https://www.npmjs.com/package/@clr/ui still shows 0.11.16.

![image](https://user-images.githubusercontent.com/1426805/39872539-4eaeb52e-5436-11e8-8dc5-a85033e11d84.png)

Also `npm install @clr/ui@0.11.17` fails

![screen shot 2018-05-10 at 9 39 16 am](https://user-images.githubusercontent.com/1426805/39872448-03a3a59e-5436-11e8-9348-0a09c1af6448.png)

#### How can the CLI team reproduce the problem?

run `npm install @clr/ui@0.11.17` and you will see the error.

### supporting information:

 - `npm -v` prints: 5.8.0
 - `node -v` prints: v8.11.1
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: OS X
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by DanProudfoot** [comment]
I'll add i'm also having this problem, and I'm also running with an @ scoped package.

**Comment by adibwoy** [comment]
Looks like it is working now:

![image](https://user-images.githubusercontent.com/1426805/39873927-4f28e0fc-543a-11e8-9c95-cf3e63b57a1f.png)

**Comment by lafiosca** [comment]
I'm experiencing this problem right now as well with a private package, as mentioned in the thread on #20448

**Comment by schipiga** [comment]
Hi I have the same problem with my project [glace-testgen](https://github.com/glacejs/glace-testgen). I published 1.1.2 and then 1.1.3 versions. But both are not available for install then

**Comment by sdovenor** [comment]
Also seeing this issue publishing to scoped private packages.

**Comment by worldviewer** [comment]
I'm having same problem myself, actually.

**Comment by xemasiv** [comment]
Also having this right now.

Tried bumping minor version and tried republishing,
the new version gets pushed but neither npm or yarn can fetch it.

:hamburger:

**Comment by schipiga** [comment]
Looks like sync problem, because after 1h my packages are available

**Comment by adibwoy** [comment]
Yes. It seems like there is a delay between the publish and when you can actually consume the packages you published. For us the packages were available 30 mins after the publish.

**Comment by lafiosca** [comment]
Took about 15-20 minutes for ours.

**Comment by ghost** [comment]
i'm experiencing the same thing on my package 

[https://www.npmjs.com/package/flavor-scss](https://www.npmjs.com/package/flavor-scss)

v1.0.10 is available in the website
but 

npm info flavor-scss

answers with a 1.0.8 as the latest version

1.0.10 was published more than one our ago.

Sigh...

**Comment by ph3n0m666** [comment]
Yup same as myself, thought it was something I had done wrong, but increment the version (of my own package) from

0.1.0
1.0.0
1.0.1
1.0.2
1.0.3

Latest was still 0.1.0 - but its been over an hour or so - like above, received email confirmation that it was published, but checking on npmjs website, latest package still refers to 0.1.0

And installing via yarn or npm returns only latest as 0.1.0 - go figure :)

**Comment by raulrene** [comment]
Same here. My packages haven't updated for the last hour. `npm view` shows an old version, so does the website. 

I'm currently browsing through their issues and came across [this thread](https://github.com/npm/npm/issues/20709) in which someone states that the npm registry is up and down. Probably some technical issues.

**Comment by DanielRuf** [comment]
> Same here. My packages haven't updated for the last hour. npm view shows an old version, so does the website.

https://github.com/npm/npm/issues/20711

Also see https://status.npmjs.org/

**Comment by ghost** [comment]
it looks like a server redundancy issue, i'm not so expert but continuously reloading

https://registry.npmjs.org/flavor-scss/

sometimes i receive 

{"_id":"flavor-scss","_rev":"106-22064bf6e7d6e92747722d8735bba43c","name":"flavor-scss","dist-tags":{"latest":"1.0.10"}, ......

somewhen else 

{"_id":"flavor-scss","_rev":"103-d57ad14fe27b36330b28d807895fdb35","name":"flavor-scss","dist-tags":{"latest":"1.0.8"}, .......

**Comment by ersel** [comment]
We're experiencing the same issue here as well. We've just published a new version, got a confirmation email from npm but the package version is wrong on the npm website.

**Comment by legodude17** [comment]
Ref #20714 and https://status.npmjs.org/incidents/3rjzvts6v98k

**Comment by nhardman** [comment]
I am still getting the problem with installing the wrong version. Yesterday I published ibm-openapi-support at version 0.0.11 and that is showing on the npm website. I just did npm install ibm-openapi-support in my terminal session and npm installed version 0.0.10.

**Comment by ghost** [comment]
fixed here.

**Comment by mjancarik** [comment]
+1
I have same problem. I published some versions without error 10 hours ago but old version still shows, both on the website and through npm view

What can I do?
